cobalt/testing/failing_tests_filter.json

```diff
@@ -0,0 +1,106 @@
+{
+  "filters": [
+    "-FileUtilTest.ChangeDirectoryPermissionsAndEnumerate",
```

Comment:
Quick nit: Can you sort this file alphabetically please ? ( in VScode, select lines, cmd+p "sort" )

Reply:
Done.